src/LibraryViewExtensionMSWebBrowser/ViewExtensionEntryPoint.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
```

Comment:
How about renaming the file to `LibraryViewExtensionMSWebBrowser.cs` or simply `ViewExtension.cs`?

Reply:
renamed